future-apps/staking/contracts/Staking.sol

```diff
@@ -158,12 +188,14 @@ contract Staking is ERCStaking, AragonApp {
   }
 
   function moveTokens(address from, address to, uint256 amount) authP(GOD_ROLE, arr(from, to, amount)) external {
-    require(accounts[from].amount >= amount);
+    // move 0 tokens makes no sense
+    require(amount > 0);
 
-    accounts[from].amount -= amount;
-    accounts[to].amount += amount;
+    // make sure we don't move locked tokens, to avoid inconsistency
+    require(unlockedBalanceOf(from) >= amount);
```

Comment:
I understand where you are coming from, but I think move tokens should be able to move them regardless of locks.

Reply:
But the (correct me if I'm wrong) the following situation may happen: an account could have more locked tokens than staked. So that account could try to stake more tokens for whatever reason but still wouldn't have available staked tokens. Even more, `unlockedBalanceOf` could revert trying to do the subtraction. I think that leaving such an inconsistent state where locked tokens amount can be greater than staked amount can give a lot of problems and be very confusing for the user. Let me put it this way: how can you have locked tokens that you don't actually have?
Another option would be to allow `moveTokens` to unlock first, but then it should require `UNLOCK_ROLE`. Maybe we can do an `unlockAndMove` function.